app/start.ps1

```diff
@@ -65,7 +65,7 @@ Write-Host "Starting backend"
 Write-Host ""
 Set-Location ../backend
 
-Start-Process -FilePath $venvPythonPath -ArgumentList "-m quart --app main:app run --port 50505 --reload" -Wait -NoNewWindow
+Start-Process -FilePath $venvPythonPath -ArgumentList "-m quart --app main:app run --port 50505 --host localhost --reload" -Wait -NoNewWindow
```

Comment:
Will this work alright in codespaces?

Reply:
if you don't specify localhost the redirect uri will not work. So perhaps we should add a disclaimer this won't work in codespaces at all.

Reply:
@pamelafox is there an env var i can check to avoid specifying this in codespaces?

Reply:
testing the fix for this

Reply:
investigated - this is actually running the app so localhost is ok. Manual test of codespaces to see if hosting still works